website/docs/theming/overview.mdx

```diff
@@ -34,92 +34,96 @@ The most common style modifiers are:
   schemes. For example, an outline button with a red color scheme.
 - **Color mode.** A component can change its visual styles based on color mode
   (light or dark).
+  
```

Comment:
Not sure if it's an appropriate way to look at theming but it feels like you can theme "locally" or "globally".

Reply:
I like this distinction. @segunadebayo thoughts?

Reply:
Yes, it make sense